includes/REST_API/Products_Controller.php

```diff
@@ -155,7 +155,15 @@ public function get_items( $request ) {
 		$query             = $this->shopping_vendors->get_vendor_class( $shopping_provider );
 
 		if ( ! $query ) {
-			return new WP_Error( 'unable_to_find_class', __( 'Unable to find class', 'web-stories' ), [ 'status' => 400 ] );
+			/** @phpstan-ignore-next-line -- this will be a shopping provider with 'none' as the default  */
+			$shopping_provider = (string) $this->settings->get_setting( Settings::SETTING_NAME_SHOPPING_PROVIDER );
+			if ( 'none' === $shopping_provider ) {
+				return new WP_Error( 'rest_shopping_provider', __( 'No shopping provider set up.', 'web-stories' ), [ 'status' => 400 ] );
+			}
+			$query = $this->shopping_vendors->get_vendor_class( $shopping_provider );
+			if ( ! $query ) {
```

Comment:
This is already done 10 lines above 🤔

Reply:
No it isn't. If you filter the vendor, but the class is not found. Say you add a custom vendor, like "jonny store". If you set the vendor to "jonny store" then deactivate the plugin that adds this vendor, the class would not be found.